movement_primitives/dmp/_base.py

```diff
@@ -15,3 +15,24 @@ def reset(self):
         self.last_t = None
         self.current_y = np.copy(self.start_y)
         self.current_yd = np.copy(self.start_yd)
+
+    def get_weights(self):
```

Comment:
* I would still prefer using another mixin for this.
* Also the members forcing_term and n_weights_per_dim are not defined.
* A docstring should not contain information about subclasses.